### ISSUE TITLE
parameter group names should default to null

### ISSUE THREAD

**Issue by aarcro** [terraform-aws-modules/terraform-aws-rds-aurora#68]
If these values are `null` AWS will create and pick the correct default group for the engine type being deployed.

```
variable "db_parameter_group_name" {
  description = "The name of a DB parameter group to use"
  type        = string
  default     = null
}

variable "db_cluster_parameter_group_name" {
  description = "The name of a DB Cluster parameter group to use"
  type        = string
  default     = null
}
```

**Comment by pierresteiner** [comment]
We tested it and it works fine. We'd also be in favor of this change

**Comment by antonbabenko** [comment]
Thanks @aarcro and @pierresteiner for opening and fixing this issue.

I didn't know that AWS would create and pick the correct default group if `null` is specified. 

I've just released [v2.9.0](https://github.com/terraform-aws-modules/terraform-aws-rds-aurora/releases/v2.9.0) where this issue is fixed.
